src/gromacs/nblib/molecules.cpp

```diff
@@ -107,19 +107,14 @@ int Molecule::numParticlesInMolecule() const
     return particles_.size();
 }
 
-void Molecule::addHarmonicBond(HarmonicType harmonicBond)
-{
-    harmonicInteractions_.push_back(std::move(harmonicBond));
-}
-
 void Molecule::addExclusion(const int particleIndex, const int particleIndexToExclude)
```

Comment:
Suggest to have them as particle1 and particle2 or particleI and particleJ. there is nothing special about first or second particle.

Reply:
I would like not to touch this on this PR, because it is about adding the bonds...
We can touch it on the Exclusions PR to the molecules that is to come...
What do you think?